doc/dsdr/0006-overlapping-episodes.md

```diff
@@ -0,0 +1,25 @@
+# Overlapping Episodes
+
+## Context
+
+There are often overlapping episodes as we merge the SSDA903 reports
+from one year to the next
```

Comment:
I wonder if this could do with a little more information about the SSDA903, as more so than "often", episodes lasting more than a year (many) will intrinsically overlap across years. And a full stop please :)